AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -18,6 +18,8 @@ amp-reddit-com.cdn.ampproject.org##.AppSelectorModal__body
 reddit.com#$#.XPromoPopup { display: none!important; }
 reddit.com#$#body.scroll-disabled { overflow: visible!important; position: static!important; }
 !
+||ssl.pstatic.net/static/m/comic/im/app_banner_
+m.comic.naver.com##img.app_banner
```

Comment:
I think cosmetic rule is enough and it's better to add this domain to the existing rule 
`gran-turismo.com,m.tv.naver.com##.app_banner`

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/77123/commits/c61616e25c2cb725435b501a5756b14d3aeb5a27